spec/components/dashboard/in_progress_collection_component_spec.rb

```diff
@@ -28,9 +28,9 @@
 
   context 'when presenter has one or more in progress collections' do
     before do
-      create(:collection_version)
-      create(:collection_version)
-      create(:collection_version)
+      create(:collection_version_with_collection)
+      create(:collection_version_with_collection)
+      create(:collection_version_with_collection)
```

Comment:
The dashboard now has links to the collection_version edit page and so the tests needs a fully formed collection.